docs/pages/concepts/why-liveblocks.mdx

```diff
@@ -0,0 +1,118 @@
+---
+meta:
+  title: "Why Liveblocks"
+  parentTitle: "Concepts"
+  description:
+    "Liveblocks is a real-time collaboration infrastructure for building
+    performant collaborative experiences."
+---
+
+Everything we do at Liveblocks comes from the strong belief that flexible office
+policies are here to stay, and as a result, every SaaS product will eventually
+be real-time collaborative to help people feel connected with one another and
+work better together.
+
+## Why collaboration
+
+In the mid 2000s and after years of development, Google launched their
+browser-based real-time collaborative Microsoft Word competitor. This approach
+completely changed the way people worked together as it enabled them to
+collaborate efficiently in real-time but also asynchronously via comments. No
+more disjointed email threads where files would get passed around. As they were
+getting more markershares, Microsoft had no other choice but to build similar
+collaboration features right into Microsoft Word and the broader Microsoft
+Office suite in order to compete.
+
+<Figure highlight>
+  <Image
+    src="/images/docs/concepts/why-liveblocks-google.png"
+    alt="Why Liveblocks - Google"
+    layout="responsive"
+    width={768}
+    height={440}
+    quality={90}
+  />
+</Figure>
+
+About a decade later, Figma flipped the whole software design industry upside
+down with their browser-based multiplayer screen design tool, and subsequently
+leaving leading companies in the space on the sideline. Very few designers
+believed a quality design tool could be built into the browser, and no designers
+really wanted to have their peers and managers looking over their shoulders
+while designing. Yet, they proved everybody wrong and a majority of designers
+started to use Figma. And because Figma was built with collaboration at its
+core, engineers, product managers, executives, marketers, copywriters quickly
+followed.
```

Comment:
```suggestion
Around a decade later, Figma flipped the whole software design industry on
its head with their browser-based tool for multiplayer design, and subsequently
left leading companies behind. Few designers
believed a quality design tool could be built into the browser, and no designers
wanted to have their peers and managers looking over their shoulders
while designing. Yet, they proved everybody wrong and a majority of designers
started to use Figma. And because Figma was built with collaboration at its
core, engineers, product managers, executives, marketers, and copywriters quickly
followed.
```
Style changes. 

I'm not sure who "they" and "everybody" is in this phrase: "they proved everybody wrong", I think this needs to be clearer